modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/config/Config.scala

```diff
@@ -167,7 +167,11 @@ object Config {
       }
     }
 
-    final case class Azure(blobStorageEndpoint: URI, messageQueue: Azure.Kafka) extends Cloud
+    final case class Azure(
+      blobStorageEndpoint: URI,
+      messageQueue: Azure.Kafka,
+      azureVaultName: Option[String]
```

Comment:
Is this a new top level config option for azure loaders? I can't see it in the min/extended configs, should we add it there?

Reply:
I missed that one. It should be added to example configs.